src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicCost = 40;
+  const minSale = 20;
+  const maxSale = 50;
```

Comment:
👍 
Nice to create variables for reusable data.
You could also create another one for (days * basicCost)